tee-worker/service/src/main_impl.rs

```diff
@@ -65,6 +65,9 @@ use substrate_api_client::{
 };
 use teerex_primitives::{Enclave as TeerexEnclave, ShardIdentifier};
 
+#[cfg(feature = "attesteer")]
+use crate::prometheus_metrics;
+
```

Comment:
IMO we should directly change
```
	let events = prometheus_metrics::fetch_marblerun_events(marblerun_base_url)
```

to 

```
	let events = crate::prometheus_metrics::fetch_marblerun_events(marblerun_base_url)
```

`mod prometheus_metrics` isn't guarded by feature, we shouldn't do that either in the derived file